README.md

```diff
@@ -1 +1,118 @@
 # Team-A-Agile
+
+## Table of Contents
+- Setup Instructions
+    - Backend Setup
+        - Java SDK
+        - Intellij
+    - Frontend Setup
+        - Microsoft Visual Studio
+- Testing Instructions
+    - Unit Tests
+    - Cucumber Tests
+
```

Comment:
Add commands/steps of how to run backend, frontend & what links to view them (http://localhost:8080/api and http://localhost:6555/)